scripts/release.py

```diff
@@ -280,14 +330,24 @@ def main(args: argparse.Namespace) -> None:
     write_version_file(version)
     write_version_to_pyproject(version)
 
-    generate_changelog(version)
-    base = git_current_branch()
-    branch = create_release_branch(version)
+    if not version.pre:
```

Comment:
this check will mean that we won't generate a changelog for release candidates. I think that's what we want, otherwise it would mean that the release after a release candidate does not have all the changelog items (or maybe there is a way to do that with towncrier, but I'm not sure)

Reply:
yes agreed, I think that makes sense :+1: